doc/source/installation.rst

```diff
@@ -1,130 +1,126 @@
 .. -*- coding: utf-8 -*-
 
 ====================
-Installation
+Installation Guide
 ====================
 
-The latest versions of **MDAnalysis** can be installed using `conda` or `pip`.
-Currently, the conda releases only support serial calculations.
-If you plan to use the parallel OpenMP algorithms, you need to
-install MDAnalysis with pip and have a working OpenMP installation.
+The latest version of **MDAnalysis** can be installed using:
+`pip <https://pip.pypa.io/en/latest/index.html>`_ (most common, for general users),
+`conda <https://docs.conda.io/projects/conda/en/latest/>`_ (recommended for better dependency management),
+or from `source <https://github.com/MDAnalysis/mdanalysis/>`_ (for development and contributing). Currently, 
+the `conda <https://docs.conda.io/projects/conda/en/latest/>`_ releases only support serial calculations. If you plan to use the parallel OpenMP algorithms, 
```

Comment:
Since you're linking conda and pip multiple times on this page, it might be easier to [separate the text and address:](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links)

```
the `conda`_
< rest of text >

.. _conda: https://docs.conda.io/projects/conda/en/latest/>
```

Reply:
Although actually it would be more informative to link to the subsections on this page using [references](https://www.sphinx-doc.org/en/master/usage/referencing.html#role-ref)!